js_modules/dagster-ui/packages/ui-core/src/code-location/CodeLocationGraphsList.tsx

```diff
@@ -0,0 +1,113 @@
+import {useQuery} from '@apollo/client';
+import {Box, NonIdealState, SpinnerWithText} from '@dagster-io/ui-components';
+import {useMemo} from 'react';
+
+import {CodeLocationSearchableList, SearchableListRow} from './CodeLocationSearchableList';
+import {PythonErrorInfo} from '../app/PythonErrorInfo';
+import {useBlockTraceOnQueryResult} from '../performance/TraceContext';
+import {WORSKPACE_GRAPHS_QUERY} from '../workspace/WorkspaceGraphsQuery';
+import {extractGraphsForRepo} from '../workspace/extractGraphsForRepo';
+import {repoAddressAsHumanString} from '../workspace/repoAddressAsString';
+import {repoAddressToSelector} from '../workspace/repoAddressToSelector';
+import {RepoAddress} from '../workspace/types';
+import {
+  WorkspaceGraphsQuery,
+  WorkspaceGraphsQueryVariables,
+} from '../workspace/types/WorkspaceGraphsQuery.types';
+import {workspacePathFromAddress} from '../workspace/workspacePath';
+
+interface Props {
+  repoAddress: RepoAddress;
+}
+
+export const CodeLocationGraphsList = (props: Props) => {
+  const {repoAddress} = props;
+
+  const selector = repoAddressToSelector(repoAddress);
+
+  const queryResult = useQuery<WorkspaceGraphsQuery, WorkspaceGraphsQueryVariables>(
+    WORSKPACE_GRAPHS_QUERY,
+    {variables: {selector}},
+  );
```

Comment:
Does the WorkspaceContext have this same data? I believe it does, in which case we should just pull this data from there using `useRepository()`

Reply:
(that is only rely on the WorkspaceContext for this data, which should update itself automatically if the code location changes)

Reply:
If not, then I think we should add the extra fields to the WorkspaceContext (though we should make sure they're not super expensive / rely on non-definition level data). cc @alangenfeld

Reply:
It doesn't look like we query for `usedSolids`. I'm also not sure we should add that to the workspace query, since this is will be a fairly low-traffic page and I don't think we need the `usedSolids` data elsewhere in the app. This page will be (fairly buried) under `Deployment > Code location > Graphs`, and serves mostly as a browsing tool.